pages/linux/smbpasswd.md

```diff
@@ -1,20 +1,21 @@
 # smbpasswd
 
-> Change a user's SMB password.
-> Samba users must also have a local Unix account.
+> Add/remove a Samba user or change their SMB password.
+> Samba users must have an existing local Unix account.
+> More information: <https://manned.org/smbpasswd.8>.
```

Comment:
```suggestion
> More information: <https://manned.org/smbpasswd>.
```

Reply:
https://manned.org/smbpasswd.5 also exists. It not having `.8` okay?

Reply:
Ahh :( sorry for this .. yep having an 8 is absolutely fine then

Reply:
we should use mfrw's suggestion.